Source/Lib/Codec/EbPacketizationProcess.c

```diff
@@ -127,7 +127,9 @@ void* PacketizationKernel(void *inputPtr)
         
         sliceType = pictureControlSetPtr->sliceType;
         
-        if(pictureControlSetPtr->pictureNumber == 0 && sequenceControlSetPtr->staticConfig.codeVpsSpsPps == 1) {
+        if((sequenceControlSetPtr->staticConfig.codeVpsSpsPps == 1) &&
```

Comment:
some SEI headers parameters are inserted when (sequenceControlSetPtr->staticConfig.codeVpsSpsPps == 1)  && (sliceType == EB_I_PICTURE) && (sequenceControlSetPtr->intraRefreshType == IDR_REFRESH) instead of only picture_number==0, will this change impact BRC case?

Reply:
> 
> 
> some SEI headers parameters are inserted when (sequenceControlSetPtr->staticConfig.codeVpsSpsPps == 1) && (sliceType == EB_I_PICTURE) && (sequenceControlSetPtr->intraRefreshType == IDR_REFRESH) instead of only picture_number==0, will this change impact BRC case?

It will. And the patch is to be updated, with a headers insertion interval setting.